plugins/sr/index.js

```diff
@@ -0,0 +1,26 @@
+module.exports = () => ({ addUtilities }) => {
+  const sr = {
+    '.sr-only': {
+      border: '0',
+      clip: 'rect(0 0 0 0)',
+      height: '1px',
+      margin: '-1px',
+      overflow: 'hidden',
+      padding: '0px',
+      position: 'absolute',
+      width: '1px'
+    },
+    '.sr-undo': {
+      clip: 'auto',
+      height: 'auto',
+      margin: '0',
+      overflow: 'visible',
+      position: 'static',
+      width: 'auto'
+    }
+  }
+
+  addUtilities(sr, {
+    variants: []
+  })
+}
```

Comment:
Megan just did a PR for this so just a little change: https://github.com/vigetlabs/FedPack/pull/118/files#diff-1fc4cc87d8d2bfb80cfe0eaac9ab3927R23